### PR TITLE
Cockpit 260

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-260.html

### CODE REVIEW

**Comment by garrett** [comment]
I'm including the doc change from #481 so regeneration is possible, as cockpit-podman has a pending PR with an empty body (no description).

When #481 lands, it should fall out from this PR.

File: _posts/2022-01-05-cockpit-260.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Cockpit 260
+author: garrett
+date: '2022-01-05'
+tags: cockpit, podman
+slug: cockpit-260
+category: release
+summary: ''
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:8]: missing

File: _posts/2022-01-05-cockpit-260.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 260 and cockpit-podman 39:
+
+
+## bridge: Warning on missing cockpit-system package
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:17]: Let's reorder this a bit: I'd say the "Action required" first, then the two Client improvements, then the minor stuff

File: _posts/2022-01-05-cockpit-260.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+This issue was assigned [CVE-2021-3698](https://access.redhat.com/security/cve/CVE-2021-3698).
+
+## Client: Show previously used hosts
+
+Cockpit client now shows list of hosts that were used before so you can log in with one click. It is also possible to remove unwanted hosts from this list.
+
+![Screenshot from 2021-12-10 08-09-06]({{ site.baseurl }}/images/260-client-show-previously-used-hosts.png)
+
+
+## Create container in pod
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:44]: "Podman:" prefix

File: _posts/2022-01-05-cockpit-260.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+It is now possible to create a container in an existing pod using the `Create container in pod` button.
+
+![screenshot of ## create container in pod]({{ site.baseurl }}/images/260-create-container-in-pod.png)
+
+Create container dialog now shows in which pod the container is created
+
+![screenshot of ## create container in pod]({{ site.baseurl }}/images/260-create-container-in-pod-2.png)
+
+## Podman restart policy
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:54]: "Podman: Set restart policy" perhaps, for consistent format?

File: _posts/2022-01-05-cockpit-260.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+For system containers it is now possible to set a restart policy for new containers.
+
+![screenshot of ## podman restart policy]({{ site.baseurl }}/images/260-podman-restart-policy.png)
+
+## Allow inserting multiple environment variables
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:60]: "Podman:" prefix

File: _posts/2022-01-05-cockpit-260.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+Cockpit 260 and cockpit-podman 39 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/260)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/39)
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-podman)
+* [cockpit-podman Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit-podman)
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:76]: podman release finished, so these URLs are available now.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks!

File: _posts/2022-01-05-cockpit-260.md (hunk 2 above)
Comment (garrett) [reply _posts/2022-01-05-cockpit-260.md:17]: Agreed; the order should be changed.

File: _posts/2022-01-05-cockpit-260.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+## Try it out
+
+Cockpit 260 and cockpit-podman 39 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/260)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:73]: [260 cockpit release](https://bodhi.fedoraproject.org/updates/?packages=cockpit) done as well now, so URLs can be updated.

File: _posts/2022-01-05-cockpit-260.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+summary: 'warning messages in cockpit-bridge, port specification for cockpit-client, restart policy in Podman, certificate login validation, previously used hosts for cockpit-client, create container in pod, specifying multiple environment variables in Podman'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 260 and cockpit-podman 39:
+
+
+![screenshot of warning on missing cockpit-system package]({{ site.baseurl }}/images/260-bridge-warning-on-missing-cockpit-system-package.png)
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:17]: This needs to be moved down.

File: _posts/2022-01-05-cockpit-260.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
+---
+title: Cockpit 260
+author: garrett
+date: '2022-01-05'
+tags: cockpit, podman
+slug: cockpit-260
+category: release
+summary: 'warning messages in cockpit-bridge, port specification for cockpit-client, restart policy in Podman, certificate login validation, previously used hosts for cockpit-client, create container in pod, specifying multiple environment variables in Podman'
```
Comment (martinpitt) [inline _posts/2022-01-05-cockpit-260.md:8]: Too long.

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: _posts/2022-01-05-cockpit-260.md (hunk 8 above)
Comment (martinpitt) [reply _posts/2022-01-05-cockpit-260.md]: Fixed.

File: _posts/2022-01-05-cockpit-260.md (hunk 9 above)
Comment (martinpitt) [reply _posts/2022-01-05-cockpit-260.md]: Fixed

**Review by martinpitt — APPROVED** [review]
Thanks!

**Comment by martinpitt** [comment]
Tweeted: https://twitter.com/CockpitProject/status/1479056394246578177

**Pull request merged**